### DISCUSSION TITLE
Group 4 Discussion - Internal services

### DISCUSSION THREAD

**Discussion by patrickpatrickpatrick** [alphagov/govuk-design-system#4335] [Community events]
Discussion for Group 4 of the Design System Hack Day - Linking Design Systems

**Comment by RosBoyes** [comment]
Topic from [Padlet](https://govukdesignsystem.padlet.org/ImHuYorks/linking-ds-hackathon-zf5uay5rigx1rq0w):

**Patterns for internal services**

These can include page layouts that are more intricate, contain more than one thing, and are suited to users with a certain level of expertise.

**Comment by RosBoyes** [comment]
Discussions from the Zoom group: 

- Would be useful to have guidance on how components and patterns might change depending on user type, e.g. hints given or limitations placed on public users might not be needed for internal/professional users. 

- Would this be an extension of existing design systems?

- Guidance on styles - e.g. at HMCTS in some places we use less spacing and smaller fonts for admin users.

- We talked about how we're often limited in what testing we do with internal users, whether that's due to the business focusing user research resources on public users or a lack of access to users for different reasons.

- Potential thinking needed around evidence or our assumptions of why existing design systems don't/won't work for internal users

- Are there patterns/components depts wouldn't share due to security concerns?

**Reply by andreamunteanu** [reply]
We also discussed how relying on training videos or annotated guides for internal users often doesn’t address the core issues they face.

**Comment by RosBoyes** [comment]
Assumptions around why public-centred design systems don't work internally:

- Internal users are doing repetitive tasks (unlike public users who may interact with a service very rarely e.g. every ten years for a passport)
- Internal users don't need as much "hand holding", they're often working at pace
- 'One thing per page' less relevant and busier screens are not necessarily a problem as users may need to complete multiple actions at once whilst also needing contextual information at the same time (e.g. case details, correspondence to action)
- We may know the devices internal users have, so we can adapt designs to suit their kit rather than designing for all eventualities (e.g. mobile versions might not be needed where we know users have to use computers attached to networks, all users may have two monitors, etc.)

**Comment by ImranH-GDS** [comment]
Hello, we have about 30 minutes left. Can each group/ subgroup think about who will play back your ideas during the show and tell?

**Comment by RosBoyes** [comment]
A question I have is, do we think this could be contextual advice as part of existing design systems... or is that risky and it needs to be kept separate? Internally at HMCTS, our admin/professional design patterns are in a separate library.

**Comment by willrichards** [comment]
Some notes on discussions to define who an internal user might be:
* People who work in departments
* Professional vs Public
* Something you are using as part of your job
* Frequency of use
* Institutional knowledge of the organisation
* Support / induction / training

Questions to explore:
* what defines internal vs external service?
* given the same standards apply internally and externally, is there a need to separate internal and external design patterns and components?
